### DISCUSSION TITLE
Is it possible to run yazi command directly? Without using keymap?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [sxyazi/yazi#1362] [1. Q&A] [answered]
There are several commands available in yazi, which you can bind. Is it possible to run them within yazi, without creating a key binding?

**Comment by sxyazi** [comment, marked as answer]
Yeah you can, check out this plugin https://github.com/KKV9/command.yazi

You can find more awesome plugins at https://yazi-rs.github.io/docs/resources
